example/material_card_example/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: material_card_example
+author: Dart Team <misc@dartlang.org>
+environment:
+  sdk: '>=2.0.0-dev.64.2 <3.0.0'
+dependencies:
+  angular: ^5.0.0-beta+1
+  angular_components: 0.9.0-beta+1
+  angular_gallery_section:
+    path: ../../angular_gallery_section
+  build_config: '>=0.2.6 <0.4.0'
```

Comment:
In this and other `pubspec.yaml` files, do you really need this to be a range? Won't ^0.3.1 do?

Reply:
It should work, but I didn't necessarily need the upgrade and didn't know if that would lock out a hypothetical user who hasn't moved past 0.2.6 yet. I admit I don't have an example of where this would be a problem.